elevenclock/settings.py

```diff
@@ -16,9 +16,9 @@
 
 try:
     import psutil
-    importedPsutil = True
+    from psutil._common import bytes2human
 except ImportError:
-    importedPsutil = False
+    print(ImportError)
```

Comment:
I am no Pythoner, i don't know that is right :)

Reply:
This must be some IDE-autogenerated line, it's not needed but does not harm